client/src/components/Ratings.jsx

```diff
@@ -4,15 +4,48 @@ class Ratings extends React.Component {
     super(props);
   }
 
+  createVisualStars(ratingValue) {
+    // CURRENT FUNCTIONALITY: 
+    // WILL RENDER CORRECT NUMBER OF FULL STARS
+    // WILL RENDER A FULL STAR IF THERE'S A HALF STAR
+    // NEED TO ADD CLASSES TO THE FULL STARS AND THE HALF STARS SO I CAN DRAW THEM SEPERATELY
+
+    const numberOfFullStars = Math.floor(ratingValue);
+
+    let fullStars = '';
+    for (let i = 0; i < numberOfFullStars; i += 1) {
+      fullStars += '★';
+    };
+
+    let halfStar = '';
+    if ((ratingValue % 1) !== 0) {
+      halfStar += '★'
+    };
+
+    // stars now have their own individual classes, find out how to append the half star to the end of the full star.
+    return (
+      <div>
+        <div className="full-star">
+          {fullStars}
+        </div>
+        <div className="half-star">
```

Comment:
Interesting. Just out of curiosity, what was the basis for creating the half-star class?

Reply:
@krise3 I'm going to use those to modify how the CSS looks. full-star will look different than half-star.